commcare_connect/templates/account/terms-and-conditions-modal.html

```diff
@@ -0,0 +1,70 @@
+<div class="fixed inset-0 z-30 bg-slate-950/50" x-cloak></div>
```

Comment:
Do we have actual terms and conditions? This is clearly not a real terms and conditions template

Reply:
This a template currently. I will ask Rama about the Terms and Conditions that we need for Connect.